draft-ietf-acme-acme.md

```diff
@@ -2982,6 +2986,12 @@ ensuring that an attacker with access to the validation channel can't obtain
 illegitimate authorization by acting as an ACME client (legitimately, in terms
 of the protocol).
 
+We do not protect against other types of abuse by a MitM on the ACME channel. 
```

Comment:
s/We/ACME/ ?

Reply:
Yeah, "ACME does not protect" is probably better.